tidy3d/components/source.py

```diff
@@ -320,11 +320,13 @@ def plot(  #  pylint:disable=too-many-arguments
     ) -> Ax:
 
         # call the `Source.plot()` function first.
-        ax = super().plot(x=x, y=y, z=z, ax=ax, **patch_kwargs)
+        ax = Box.plot(self, x=x, y=y, z=z, ax=ax, **patch_kwargs)
```

Comment:
I'm worried this won't work anymore if the patch_kwargs contain "arrow_base". Maybe we need to do something like 

```py
kwargs_arrow_base = patch_kwargs.pop("arrow_base")
```
before this line?
I could be wrong though.

Reply:
It seems to work, I'm able to reproduce the plots, but I made the change anyway to be safe.

Reply:
It seems to work, I'm able to reproduce the plots, but I made the change anyway to be safe.